docs/0.1/creating_locations.md

```diff
@@ -194,15 +193,15 @@ Set the following Grid environment variables to specify information for the
    reference the ***user key files*** in `$HOME/.grid/keys`, not the Grid
    daemon's key files in `/etc/grid/keys`.
 
-1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+3. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
```

Comment:
using all "1"s here was intentional. This is automatic numbering so it requires fewer changes for any further updates to the doc.

If you want to start at a specific number you use the notation `{:start="5"}` and have the first number be `5` followed by `1`s